docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+version: "3.3"
+services:
+  star-tides:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    restart: always
+    volumes:
+      - ./:/app
+    depends_on:
+      - mongodb_container
+    ports:
+      - 5000:5000
+  test:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    restart: always
+    volumes:
+      - ./:/app
+    depends_on:
+      - mongodb_container
+    ports:
+      - 5001:5000
+    command: bash -c "cd /app && pytest -q star_tides/tests/tests"
```

Comment:
Also run Angular tests?

Reply:
Decided against doing that because it requires chrome headless browser to be installed, which I don't want to add to the container.